install-bashhub.sh

```diff
@@ -102,11 +102,9 @@ download_and_install_env() {
     PYTHON=$(which $python_command)
     echo "Using Python path $PYTHON"
 
-    VERSION=20.10.0
-    VERSION_URL="https://github.com/pypa/get-virtualenv/raw/$VERSION/public/virtualenv.pyz"
```

Comment:
Using this version and the specific version url is intentional. Is the update needed to support newer versions of Python?